components/Badge.svelte

```diff
@@ -2,6 +2,11 @@
   export let color = 'gray'
   export let borderRadius = '50%'
   export let padding = '.2em'
+  export let bordered = false
+
+  let transparentColor = ''
+
+  $: if (color.length == 7) transparentColor = color + '12' //sets transparent background color. only works with full hexadecimal color
```

Comment:
What if you give it a color word that is 7 characters long?

Reply:
then the transparentColor will be invalid the background will be white